modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,58 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+The `Instrumentation` custom resource configures only OTLP exporter:
+
+.Sample exporter configuration
+[source,yaml]
+----
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      secretName: serving-certs # <2>
+      configMapName: ca-bundle  # <3>
+      ca_file: service-ca.crt # <4>
+      cert_file: tls.crt # <5>
+      key_file: tls.key # <6>
+----
+<1> OpenTelemetry protocol (OTLP) endpoint. HTTP scheme `https://` enables secure connection and requires `tls` configuration.
+<2> Secret name which contains TLS certificate and key. The CA certificate can be in the secret or in the config map. The secret must exists in the namespace of the pod injecting the auto-instrumentation.
+<3> Configmap name if the CA certificate is present in the config map. The configmap must exist in the namespace of the pod injecting the auto-instrumentation.
+<4> Ca defines the key of certificate (e.g. ca.crt) in the configmap map, secret or absolute path to a certificate. The absolute path can be used when certificate is already present on the workload filesystem e.g. `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`
+<5> Cert defines the key of client certificate (e.g. tls.crt) in the secret or absolute path to a certificate. The absolute path can be used when the certificate is already present on the workload filesystem.
+<6> Key defines the key of client key (e.g. tls.key) in the secret or absolute path to a key. The absolute path can be used when the key is already present on the workload filesystem.
```

Comment:
Are these two callouts equally applicable to both Secrets and config maps?

Reply:
cert and key can be defined only in the secret. The CA can be defined in the secret and config map. The config map has a higher precedence.